Source/WebCore/platform/mediastream/WebRTCProvider.cpp

```diff
@@ -198,6 +200,107 @@ bool WebRTCProvider::isH264EncoderSmooth(const VideoConfiguration&)
     return true;
 }
 
+static String defaultHEVCProfileCompatibilityIndicator(const String& profileId)
+{
+    auto id = parseInteger<int>(profileId);
+    if (!id || *id < 0 || *id >= 32)
+        return { };
+
+    auto numberOfZeros = *id / 4;
+    auto result = makeString(*id % 4);
+    while (numberOfZeros--)
+        result = makeString(result, "0"_s);
+
+    return result;
+}
+
+static String contentTypeFromRTPVideoMimeType(const String& mimeType)
+{
+    ContentType contentType { mimeType };
+    auto containerType = contentType.containerType();
+    if (equalLettersIgnoringASCIICase(containerType, "video/h264"_s)) {
+        auto profileLevelId = contentType.parameter("profile-level-id"_s);
+        if (profileLevelId.isEmpty())
+            profileLevelId = "42001f"_s;
+        return makeString("video/mp4;codecs=avc1."_s, profileLevelId);
+    }
+
+    if (equalLettersIgnoringASCIICase(containerType, "video/h265"_s)) {
+        auto profileSpace = contentType.parameter("profile-space"_s);
```

Comment:
The contents of this `if` statement should use `HEVCUtilities.h` to create the `codec` string by building a `HEVCParameters` object.

in fact, the parsing portion should likely also be added to `HEVCUtilities.h`.

Reply:
Sounds good, done

Reply:
As for the parsing portion, this is RTP specific, so I am not sure it is good to move them to the utilities.h files.